civis/io/_tables.py

```diff
@@ -1,5 +1,5 @@
 import csv
-import tempfile
+import io
```

Comment:
`io` is imported below (it's in the weird `try` `except` to lead the way towards python2 compatibility).